.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -3,24 +3,24 @@ name: Report a bug with Home Assistant
 about: Report an issue with Home Assistant
 ---
 <!-- READ THIS FIRST:
-- If you need additional help with this template, please refer to https://www.home-assistant.io/help/reporting_issues/
-- Make sure you are running the latest version of Home Assistant before reporting an issue: https://github.com/home-assistant/home-assistant/releases
-- Do not report issues for integrations if you are using custom components or integrations.
-- Provide as many details as possible. Paste logs, configuration samples and code into the backticks.
-DO NOT DELETE ANY TEXT from this template! Otherwise, your issue may be closed without comment.
+  - If you need additional help with this template, please refer to https://www.home-assistant.io/help/reporting_issues/
+  - Make sure you are running the latest version of Home Assistant before reporting an issue: https://github.com/home-assistant/home-assistant/releases
+  - Do not report issues for integrations if you are using custom components or integrations.
+  - Provide as many details as possible. Paste logs, configuration samples and code into the backticks.
+  DO NOT DELETE ANY TEXT from this template! Otherwise, your issue may be closed without comment.
 -->
 ## The problem
 <!-- 
-    Describe the issue you are experiencing here to communicate to the
-    maintainers. Tell us what you were trying to do and what happened instead.
+  Describe the issue you are experiencing here to communicate to the
+  maintainers. Tell us what you were trying to do and what happened instead.
 -->
 
 
 ## Environment
 <!--
-    Provide details about the versions you are using, which helps us reproducing
-    and finding the issue quicker. Version information is found in the
-    Home Assistant frontend: Developer tools -> Info.
+  Provide details about the versions you are using, which helps us reproducing
```

Comment:
which helps us to reproduce and find the issue quicker.

Reply:
Done.